src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -382,4 +382,22 @@ protected function isCsrfTokenValid(string $id, ?string $token): bool
 
         return $this->container->get('security.csrf.token_manager')->isTokenValid(new CsrfToken($id, $token));
     }
+
+    /**
+     * Dispatches a message to the bus.
+     *
+     * @param object $message The message to dispatch
+     *
+     * @return mixed The result from the bus
+     *
+     * @final
+     */
+    protected function dispatch($message)
+    {
+        if (!$this->container->has('message_bus')) {
+            throw new \LogicException('The message bus is not enabled in your application. Enable it with the "message" key in "config/packages/framework.yaml".');
```

Comment:
The message bus should be automatically enabled when the component is installed. So, the right hint is to install the message component.